scanomatic/image_analysis/image_grayscale.py

```diff
@@ -422,8 +432,9 @@ def detect_grayscale(im_trimmed, grayscale: Grayscale):
                     np.save(os.path.join(
                         Paths().log,
                         f"gs_segment_{i}.npy",
+                    ),
```

Comment:
Strange indentation, better to break before `os.path` and indent accordingly?

Reply:
Fixed